recipes/owa-env-gst/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "owa-env-gst" %}
+{% set version = "0.3.9a4" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
I see, I guess conda semantic is different from PEP440. I fixed all `owa`-related package's version to `0.3.9`.

Reply:
Conda is Python agnostic. With that said, it does understand PEP440 and we do recommend it for Python packages. CFEP-05 is not about the version string, but more of a package publication organization. We require explicit dev sub channels.